File: Modul 2/Bahasa Python/PRAK110-2210817310010-AERONTJITRADI.py
import math
s1=12
s3=5
s2=math.sqrt((s1*s1)+(s3*s3))
k=s3+s1+s2
l=(s3*s1)/2
print("Diketahui:")
print("Alas = {} cm".format(s3))
print("Tinggi = {} cm\n".format(s1))
print("Jawab:")
print("Sisi A = {} cm".format(s1))
print("Sisi B = {} cm".format(round(s2)))
print("Sisi C = {} cm".format(s3))
print("Keliling = {} cm".format(round(k)))
print("Luas = {} cm".format(round(l)))

File: Modul 2/Soal 10/PRAK110-2210817310010-AERONTJITRADI.c
#include <stdio.h>
#include <math.h>

int main(){
    float s1,s2,s3,k,l;
    s1=12;
    s3=5;
    s2=sqrt((s1*s1)+(s3*s3));
    k=s1+s2+s3;
    l=0.5*s1*s3;

    printf("Diketahui: \n");
    printf("Alas = %.f cm\n", s3);
    printf("Tinggi = %.f cm\n\n", s1);
    printf("Jawab: \n");
    printf("Sisi A = %.f cm\n", s1);
    printf("Sisi B = %.f cm\n", s2);
    printf("Sisi C = %.f cm\n", s3);
    printf("Keliling = %.f cm\n", k);
    printf("Luas = %.f cm", l);

    return 0;
}